packages/sitecore-jss-nextjs/src/utils.ts

```diff
@@ -30,6 +30,15 @@ export const getPublicUrl = (): string => {
   return url.toString().replace(/\/$/, '');
 };
 
+/**
+ * Get hostname from from the provided URL.
+ * @param {Array} [urls] - The URL to parse
+ */
+export const getHostName = (urls: [string]): Array<string> => {
+  const hostName = urls ? urls.map((url) => new URL(url).hostname) : [];
```

Comment:
Might want to wrap this in a try/catch. I believe an error is thrown by `URL` constructor if an invalid URL is passed in (add test case for this).